docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -320,16 +320,31 @@ const {
   // Optional, filter for rooms that allow entry to a user's ID in `usersAccesses`
   userId: "my-user-id",
 
-  // Optional, filter for rooms with custom metadata in `metadata`
-  metadata: {
-    myRoomType: "whiteboard",
-  },
+  // Optional, use advanced filtering
+  query: {
+    // Optional, filter for rooms with custom metadata in `metadata`
+    metadata: {
+      myRoomType: "whiteboard",
+      shared: true,
+      time: 0,
```

Comment:
```suggestion
```
Room metadata only supports `string` as a value so we can only filter with `:` and `string` for now.